iris_grib/tests/results/unit/load_cubes/load_cubes/reduced_raw.cml

```diff
@@ -18,7 +18,7 @@
         </auxCoord>
       </coord>
       <coord>
-        <dimCoord id="c9091dc6" long_name="level_pressure" points="[2.00004005432]" shape="(1,)" units="Unit('Pa')" value_type="float64"/>
+        <dimCoord id="c9091dc6" long_name="level_pressure" points="[0.0]" shape="(1,)" units="Unit('Pa')" value_type="float64"/>
```

Comment:
Here's my justification...
```
$ grib_dump -O iris-test-data/test_data/GRIB/reduced/reduced_gg.grib2 
  . . .
======================   SECTION_4 ( length=770, padding=0 )   ======================
  . . .
6-7       NV = 184
  . . .
23        typeOfFirstFixedSurface = 105 [Hybrid level (grib2/tables/5/4.5.table) ]
24        scaleFactorOfFirstFixedSurface = 0
25-28     scaledValueOfFirstFixedSurface = 1
29        typeOfSecondFixedSurface = 255 [Missing (grib2/tables/5/4.5.table) ]
30        scaleFactorOfSecondFixedSurface = MISSING
31-34     scaledValueOfSecondFixedSurface = MISSING
35-770    pv = (184,736) {
0.0000000000e+00, 2.0000400543e+00, 3.9808320999e+00, 7.3871860504e+00, 1.2908319473e+01, 2.1413612366e+01, 3.3952857971e+01, 5.1746601105e+01, 
7.6167655945e+01, 1.0871556091e+02, 1.5098602295e+02, 2.0463745117e+02, 2.7135650635e+02, 3.5282449341e+02, 4.5068579102e+02, 5.6651922607e+02, 
7.0181335449e+02, 8.5794580078e+02, 1.0361665039e+03, 1.2375854492e+03, 1.4631639404e+03, 1.7137095947e+03, 1.9898743896e+03, 2.2921555176e+03, 
2.6208984375e+03, 2.9763022461e+03, 3.3584257812e+03, 3.7671960449e+03, 4.2024165039e+03, 4.6637763672e+03, 5.1508598633e+03, 5.6631562500e+03, 
6.1998393555e+03, 6.7597270508e+03, 7.3414697266e+03, 7.9429262695e+03, 8.5646240234e+03, 9.2083056641e+03, 9.8735605469e+03, 1.0558881836e+04, 
1.1262484375e+04, 1.1982662109e+04, 1.2713897461e+04, 1.3453225586e+04, 1.4192009766e+04, 1.4922685547e+04, 1.5638053711e+04, 1.6329560547e+04, 
1.6990623047e+04, 1.7613281250e+04, 1.8191029297e+04, 1.8716968750e+04, 1.9184544922e+04, 1.9587513672e+04, 1.9919796875e+04, 2.0175394531e+04, 
2.0348916016e+04, 2.0434158203e+04, 2.0426218750e+04, 2.0319011719e+04, 2.0107031250e+04, 1.9785357422e+04, 1.9348775391e+04, 1.8798822266e+04, 
1.8141296875e+04, 1.7385595703e+04, 1.6544585938e+04, 1.5633566406e+04, 1.4665645508e+04, 1.3653219727e+04, 1.2608383789e+04, 1.1543166992e+04, 
1.0471310547e+04, 9.4052226562e+03, 8.3562529297e+03, 7.3351645508e+03, 6.3539208984e+03, 5.4228027344e+03, 4.5502158203e+03, 3.7434643555e+03, 
3.0101469727e+03, 2.3562026367e+03, 1.7848546143e+03, 1.2976561279e+03, 8.9519354248e+02, 5.7631414795e+02, 3.3677236938e+02, 1.6204342651e+02, 
5.4208335876e+01, 6.5756278038e+00, 3.1600000802e-03, 0.0000000000e+00, 0.0000000000e+00, 0.0000000000e+00, 0.0000000000e+00, 0.0000000000e+00, 
0.0000000000e+00, 0.0000000000e+00, 0.0000000000e+00, 0.0000000000e+00
... 84 more values
} # ieeefloat pv 
```
So, the level values in PV[0:] start out
0.0000000000e+00, 2.0000400543e+00, 3.9808320999e+00, 7.3871860504e+00,
and we have replaced the '2.00' one with plain old '0.00'.
Meanwhile, sigma values in PV[92:] start with
0.0000000000e+00, 0.0000000000e+00, 0.0000000000e+00, 0.0000000000e+00,
- so no changes in those.